sdk/subscription/azure-mgmt-subscription/setup.cfg

```diff
@@ -1,2 +0,0 @@
-[bdist_wheel]
```

Comment:
When is the release timeline for this package?

On Jan 1st, Python 2.7 is being deprecated from our pipelines. If you intend to release before that, you need to add this file (and it's content back in).